elastalert/test_rule.py

```diff
@@ -140,10 +140,12 @@ def mock_hits(self, rule, start, end, index):
 
         # Remove all fields which don't match 'include'
         for doc in docs:
+            remove = []
```

Comment:
Fix this variable name to describe what it contains.

Reserve action verbs for method names. This will prevent people from becoming very confused as they work with this code.